ocaml-lsp-server/src/code_actions/action_destruct.ml

```diff
@@ -15,8 +16,13 @@ let code_action_of_case_analysis doc uri (loc, newText) =
     WorkspaceEdit.create ~documentChanges:[ `TextDocumentEdit edit ] ()
   in
   let title = String.capitalize_ascii action_kind in
+  let command =
+    Command.create ~title:"Jump to First Hole" ~command:"ocaml.next-hole"
```

Comment:
"First Hole" implies to me that we are jumping to the first hole in the document. The title should match the next hole behavior.